src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
+
+  return totalCost;
```

Comment:
Consider using descriptive constants for the daily cost, long term rental threshold, and the respective discounts. This will make your code more readable and maintainable, as per checklist item 2.